examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/errno.h>
```

Comment:
Shouldn't it just be

`#include <errno.h>`

https://pubs.opengroup.org/onlinepubs/009604499/basedefs/errno.h.html

Reply:
Hmm.  "man errno" was claiming `<sys/errno.h>` at least on Mac, but it looks like just `<errno.h>` works.  I'll do a followup to adjust to that.

Reply:
https://github.com/project-chip/connectedhomeip/pull/5326